.github/workflows/ci.yml

```diff
@@ -4,6 +4,10 @@ on: [push, pull_request]
 
 jobs:
   CI:
+    # Enable pull request trigger only from forks, since otherwise we get 2 job runs in PRs
+    if:
+      github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name !=
+      github.repository
```

Comment:
I seems it's either this or revising the `on` trigger to
```
on:
  push:
    branches:
      - master
  pull_request:
```
in order to avoid double CI runs (in the later case CI doesn't get run for pushed branches without an open PR).